apps/arena/mix.exs

```diff
@@ -51,8 +51,9 @@ defmodule Arena.MixProject do
       {:exbase58, "~> 1.0.2"},
       {:credo, "~> 1.7", only: [:dev, :test], runtime: false},
       {:plug_cowboy, "~> 2.5"},
-      {:toxiproxy_ex, "~> 1.1.1"},
-      {:configurator, in_umbrella: true}
+      {:toxiproxy_ex, "~> 1.1.1"}
+      # TODO We will find a way to solve this dependency.
```

Comment:
As was stated in slack, we should delete the dependency altogether instead of adding this TODO

Reply:
Yep, Done!